portfolio-assignment/index.html

```diff
@@ -105,8 +108,12 @@
     <script src="scripts/vendor/marked.js"></script>
     <script src="scripts/vendor/highlight.pack.js"></script>
 
-    <script src="scripts/data.js"></script>
     <script src="scripts/projects.js"></script>
     <script src="scripts/projectView.js"></script>
+
+    <script>
+      //call it here for now
```

Comment:
noooooo no functions in the html. can you stick it in an index.js file and call on document ready instead?